main.go

```diff
@@ -106,6 +111,29 @@ type Config struct {
 	PprofListenOn          string            `default:"localhost:6060" desc:"pprof URL to ListenAndServe" split_words:"true"`
 }
 
+type ipamPolicy uint8
```

Comment:
Consider making it a function as we do it in the cmd-forwarder-vpp https://github.com/networkservicemesh/cmd-forwarder-vpp/blob/main/internal/vppinit/vppinit.go#L75